scripts/build-dtest-jars.sh

```diff
@@ -38,8 +38,8 @@ else
   #   "cassandra-4.0:cassandra-4.0"
   # Due to MacOS being stuck on Bash < 4, we don't use associative arrays here.
   CANDIDATE_BRANCHES=(
-    "cassandra-4.0:1f79c8492528f01bcc5f88951a1cc9e0d7265c54"
-    "cassandra-4.1:725655dda2776fef35567496a6e331102eb7610d"
+    "cassandra-4.0:af752fcd535ccdac69b9fed88047b2dd7625801e"
```

Comment:
it feels like we should use the tag here instead: `cassandra-4.0.12`
```suggestion
    "cassandra-4.0:cassandra-4.0.12"
```

Reply:
Good catch. Forgot that tags are supported.